packages/terra-form-checkbox/src/orion-fusion-theme/Checkbox.module.scss

```diff
@@ -73,6 +73,6 @@
 
     // Inline SVG
     @include terra-inline-svg-var('--terra-form-checkbox-checked-disabled-before-content', '<svg xmlns="http://www.w3.org/2000/svg"></svg>');
-    @include terra-inline-svg-var('--terra-form-checkbox-checked-before-content', '<svg xmlns="http://www.w3.org/2000/svg"></svg>');
+    // @include terra-inline-svg-var('--terra-form-checkbox-checked-before-content', '<svg xmlns="http://www.w3.org/2000/svg"></svg>');
```

Comment:
How does commenting out this line fix the issue, and does it impact anything else?

Reply:
Instead of removing !! Can we try reducing the size of svg so that additional clickable region gets shrinked as well

Reply:
Is the SVG needed, even if it's empty?